hashdist/cli/frontend_cli.py

```diff
@@ -123,7 +123,16 @@ class Develop(ProfileFrontendBase):
     """
     Builds a development profile in the HashDist YAML profile spec
     format, at the same location as the profile yaml file, but without
-    the .yaml suffix.
+    the .yaml suffix. You can use the 'target' argument to name it differently
```

Comment:
The first sentence is confusing on its own --- it seems to imply that the location will be the profile name without the .yaml suffix. Typically you already have this directory, with the regular profile. So this is very confusing, because I think the standard usage will be to provide the target name using 'target', that's at least how I do it.

So I can move it down, but then I need to rework the first sentence stating that the final directory will be 'target', and then further down just mention that if you omit 'target', then it will use your profile name.